imports/plugins/core/ui-tagnav/client/components/tagGroupBody.js

```diff
@@ -0,0 +1,173 @@
+import debounce from "lodash/debounce";
+import update from "react/lib/update";
+import React, { Component, PropTypes } from "react";
```

Comment:
Yes I understand, but the style guide only states that NPM imports come first. If we are adding any additional requirements regarding order they should be documented in the style guide. Current style guidelines:

```
import order
npm packages
meteor core packages
meteor (Atmosphere) packages
local app files
```